storage/sql/migrate.go

```diff
@@ -38,8 +38,10 @@ func (c *conn) migrate() (int, error) {
 
```

Comment:
Correct me if I'm wrong, but if this change has nothing to do with the mysql backend, I'd rather split it off... 😃

Reply:
This is because of the multistatement stuff :)